src/DIRAC/WorkloadManagementSystem/ConfigTemplate.cfg

```diff
@@ -17,6 +17,14 @@ Services
       Default = authenticated
     }
   }
+  TornadoPilotLogging
```

Comment:
There's an old DISET PilotLogging service that I would just remove (it was never used)

Reply:
I haven't removed it yet.